packages/core/types/src/http/product/admin/queries.ts

```diff
@@ -1,16 +1,14 @@
 import {
+  BaseProductListParams,
   BaseProductOptionParams,
-  BaseProductParams,
   BaseProductTagParams,
-  BaseProductTypeParams,
   BaseProductVariantParams,
 } from "../common"
 
 export interface AdminProductTagParams extends BaseProductTagParams {}
-export interface AdminProductTypeParams extends BaseProductTypeParams {}
 export interface AdminProductOptionParams extends BaseProductOptionParams {}
 export interface AdminProductVariantParams extends BaseProductVariantParams {}
-export interface AdminProductParams extends BaseProductParams {
+export interface AdminProductListParams extends BaseProductListParams {
```

Comment:
This type renaming needs to be reflected in two places in our core package:
- `maybe-apply-price-list-filter.ts`
- `api/admin/products/route.ts`

Otherwise, it won't build